openbb_terminal/stocks/comparison_analysis/yahoo_finance_view.py

```diff
@@ -226,5 +229,7 @@ def display_correlation(
 
     if not external_axes:
         theme.visualize_output()
-
+    
+    export_data(export, os.path.dirname(os.path.abspath(__file__)), "hcorr", df_similar)
```

Comment:
@alexferrari88 most likely this line is longer than 88 chars and that's why black is complaining. Run `black .` from the project root and black will check and fix formatting in all the files of the project. 
Once you commit those changes it should be good

Reply:
thank you! strangely when I have run black it said that no files were unchanged...
anyway I have manually fixed it and it seems working (at least for me!)